src/main/groovy/edu/oregonstate/mist/directoryapi/DirectoryEntity.groovy

```diff
@@ -0,0 +1,31 @@
+package edu.oregonstate.mist.directoryapi
+
+/**
+ * Directory entity representation class.
+ */
+class DirectoryEntity {
+    String firstName
+    String lastName
+    String fullName
+    String primaryAffiliation
+    String jobTitle
+    String department
+    String departmentMailingAddress
```

Comment:
Let's make a note of it. Once we have a list of questions / comments, I'll run that by other devs on campus to get feedback. LDAP does the best to figure out the department address and job information. For example, there's only one set of department related information, but people can have split appointments (more than one job) at OSU.